### PR TITLE
IBX-3175: Hide assigned content tab

### PR DESCRIPTION
**Pull request by juskora**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-3175
| Versions      | master, 4.3+

Related PR: https://github.com/ibexa/taxonomy/pull/151

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [x] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml [hunk 1]
```diff
@@ -13,4 +13,5 @@ ibexa_taxonomy:
             field_mappings:
                 identifier: category_identifier
                 parent: parent_category
-                name: name_field
\ No newline at end of file
+                name: name_field
+            assigned_content_tab: false
```
Comment (konradoboza) [inline code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml:17]:
```suggestion
            assigned_content_tab: false

```

File: docs/content_management/taxonomy/taxonomy.md [hunk 2]
```diff
@@ -36,6 +36,17 @@ To do it, first, create a new Content Type with `content_category` identifier an
 
 Next, in `config/packages/ibexa_taxonomy.yaml` add the following configuration:
 
+``` yaml
+[[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml', 1,16 )=]]
+```
+
+### Hide Content tab
+
+The **Content** tab in taxonomy objects, for example, tags and categories, lists all Content assigned to the current category.
```
Comment (konradoboza) [inline docs/content_management/taxonomy/taxonomy.md:45]: Category in a sense of our product is a kind of taxonomy, so maybe:
```suggestion
The **Content** tab in taxonomy objects, for example, tags and categories, lists all Content assigned to the current taxonomy. 
```

File: docs/content_management/taxonomy/taxonomy.md [hunk 3]
```diff
@@ -36,6 +36,17 @@ To do it, first, create a new Content Type with `content_category` identifier an
...
+``` yaml
+[[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml', 1,16 )=]]
+```
+
+### Hide Content tab
+
+The **Content** tab in taxonomy objects, for example, tags and categories, lists all Content assigned to the current category. 
+You can hide the **Content** tab in the **Categories** view.
+
+In the `config/packages/ibexa_taxonomy.yaml` add `assigned_content_tab` with the flag `false`:
```
Comment (konradoboza) [inline docs/content_management/taxonomy/taxonomy.md:48]:
```suggestion
In the `config/packages/ibexa_taxonomy.yaml` add `assigned_content_tab` with the flag `false` (for other taxonomies this flag is by default set to `true`):
```

**Review by konradoboza — APPROVED** [review]

File: docs/content_management/taxonomy/taxonomy.md [hunk 4]
```diff
@@ -36,6 +36,17 @@ To do it, first, create a new Content Type with `content_category` identifier an
...
+
+### Hide Content tab
+
+The **Content** tab in taxonomy objects, for example, tags and categories, lists all Content assigned to the current taxonomy. 
+You can hide the **Content** tab in the **Categories** view.
+
+In the `config/packages/ibexa_taxonomy.yaml` add `assigned_content_tab` with the flag `false` (for other taxonomies this flag is by default set to `true`):
+
 ``` yaml
 [[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml')=]]
```
Comment (DominikaK) [inline docs/content_management/taxonomy/taxonomy.md:51]: Select a range of lines here, maybe? Now it pretty much repeats the code block above

File: docs/content_management/taxonomy/taxonomy.md [hunk 5]
```diff
@@ -36,6 +36,17 @@ To do it, first, create a new Content Type with `content_category` identifier an
 
 Next, in `config/packages/ibexa_taxonomy.yaml` add the following configuration:
 
+``` yaml
+[[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml', 1,16 )=]]
```
Comment (DominikaK) [inline docs/content_management/taxonomy/taxonomy.md:40]: Any reason to clip the `ibexa_taxonomy` key?

Comment (juskora) [reply docs/content_management/taxonomy/taxonomy.md:40]: Oh, probably didn't render, as it's in the code sample file. Will fix it.

File: docs/content_management/taxonomy/taxonomy.md [hunk 6]
```diff
@@ -37,7 +37,18 @@ To do it, first, create a new Content Type with `content_category` identifier an
...
+
+### Hide Content tab
+
+The **Content** tab in taxonomy objects, for example, tags and categories, lists all Content assigned to the current taxonomy. 
+You can hide the **Content** tab in the **Categories** view.
+
+In the `config/packages/ibexa_taxonomy.yaml` add `assigned_content_tab` with the flag `false` (for other taxonomies this flag is by default set to `true`):
+
+``` yaml
+[[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml', 0,2 )=]][[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml', 10,17 )=]]
```
Comment (DominikaK) [inline docs/content_management/taxonomy/taxonomy.md:51]:
```suggestion
[[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml', 0,2 )=]][[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml', 9,17 )=]]
```
without it the rendering eats `content_categories`

File: docs/content_management/taxonomy/taxonomy.md [hunk 7]
```diff
@@ -37,7 +37,18 @@ To do it, first, create a new Content Type with `content_category` identifier an
 Next, in `config/packages/ibexa_taxonomy.yaml` add the following configuration:
 
 ``` yaml
-[[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml')=]]
+[[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml', 0,16 )=]]
```
Comment (DominikaK) [inline docs/content_management/taxonomy/taxonomy.md:40]:
```suggestion
[[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml')=]]
```
if it's all lines of a page, you can drop the line range.

**Review by DominikaK — APPROVED** [review]

**Pull request merged**